package.json

```diff
@@ -108,7 +108,8 @@
     "supertest": "6.2.4",
     "ts-jest": "29.0.3",
     "typescript": "4.6.2",
-    "yargs": "17.6.0"
+    "yargs": "17.6.0",
+    "knex": "2.4.0"
```

Comment:
why is knex added in the root folder package.json?

Reply:
Since I use knex directly to create a new database since I can't do that trough strapi.